neural_network/GAN.py

```diff
@@ -1,9 +1,7 @@
-import numpy as np,sys,time
```

Comment:
Let's just run `isort` on all the imports...
```
import sys

import matplotlib.gridspec as gridspec
import matplotlib.pyplot as plt
import numpy as np
from sklearn.utils import shuffle
from tensorflow.examples.tutorials.mnist import input_data
```

Reply:
https://github.com/timothycrosley/isort

Reply:
I ran the isort.
Thanks for this wonderful information.